.github/workflows/release_python.yml

```diff
@@ -77,7 +77,11 @@ jobs:
       matrix:
         include:
           - target: x86_64
+            wheel_suffix: macosx_11_0_x86_64
+          - target: arm64
+            wheel_suffix: macosx_11_0_arm64
           - target: universal2
+            wheel_suffix: macosx_11_0_x86_64.macosx_11_0_arm64.macosx_11_0_universal2
```

Comment:
Apparently the `universal2` includes the suffix for everything it includes. This is also why it takes twice as long to build because it has to compile both versions.